frappe/communications/doctype/communication/communication.js

```diff
@@ -143,7 +143,7 @@ frappe.ui.form.on("Communication", {
 					label: __("Reference Doctype"),
 					fieldname: "reference_doctype",
 					get_query: function () {
-						return { query: "frappe.email.get_communication_doctype" };
+						return { query: "frappe.communications..inbox.get_communication_doctype" };
```

Comment:
```suggestion
						return { query: "frappe.communications.inbox.get_communication_doctype" };
```

Reply:
Thanks, I double checked with `rg 'communications\.\.'` and it was the only _such_ one.